pkg/kube/controllers/extendedstatefulset/extendedstatefulset_controller.go

```diff
@@ -67,69 +63,5 @@ func AddExtendedStatefulSet(ctx context.Context, config *config.Config, mgr mana
 		return err
 	}
 
-	// Watch ConfigMaps referenced by the ExtendedStatefulSet
-	configMapPredicates := predicate.Funcs{
-		CreateFunc:  func(e event.CreateEvent) bool { return true },
```

Comment:
Why is this true for CM and false for secrets (line 104)

Reply:
It should be true both.